app/service/sorting_test.go

```diff
@@ -137,13 +137,13 @@ func TestApplySorting(t *testing.T) {
 			shouldPanic: errors.New(`unsupported type "interface{}" for field "name"`)},
 		{name: "unallowed from fields",
 			args: args{
-				urlParameters: "?from.field=Joe&from.version=2",
+				urlParameters: "?from.field=Joe&from.version=2&from.name=Jane",
 				acceptedFields: map[string]*FieldSortingParams{
 					"name": {ColumnName: "sName", FieldType: "interface{}"},
```

Comment:
don't use `FieldType: "interface{}"` here as it is not what you are testing

Reply:
fixed